bigbluebutton/greenlight
# Internationalize date format

## Description


Dates were displayed in local time but the date format itself was not internationalized/localized. This PR changes the view_date helper so that locales are used to get the date format (time.formats.default). This PR contains formats for English (the format that used to be hard coded before) and German.

## Testing Steps


Add a new date format string to conf/locales/LANG.yml, e.g. for de_DE.yml:

```
  time:
    formats:
      default: "%d.%m.%Y, %H:%M"
```

Change browser locale to the languge. Visit a page that displays a date, e.g. rooms index or settings. The dates should be shown in the date format that was configured for that language.

(This PR was done as part of my work for ColloCall/UNI:CODE IT Solutions GmbH)
